src/ConfCrypt/Template.hs

```diff
@@ -0,0 +1,70 @@
+module ConfCrypt.Template (
+    renderTemplate
+    ) where
+
+import Text.Megaparsec
+import Text.Megaparsec.Char
+import Data.Text
+import ConfCrypt.Types (Parameter(..))
+import qualified Data.Text as T
```

Comment:
This import is redundant: it is used only once to access `pack`, which is also accessed unqualified in another place.